src/Sulu/Bundle/ContentBundle/Resources/public/js/validation/types/block.js

```diff
@@ -352,15 +352,6 @@ define([
                     }.bind(this));
                 },
 
-                checkSortable: function() {
-                    // check for dragable
-                    if (this.getChildren().length <= 1) {
```

Comment:
@wachterjohannes This method was not working correctly, since only the script tags used as block types have been loaded until here. So this statement was only true when more than two block types. I just removed it, and decided to make a div sortable as long as it has a maximum higher than 1.